committee-rails.gemspec

```diff
@@ -21,5 +21,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'committee', '>= 4.4.0'
   spec.add_dependency 'activesupport'
   spec.add_dependency 'actionpack'
-  spec.add_dependency 'railties'
+  spec.add_dependency 'railties', '>= 5.1'
```

Comment:
I thought it would be more helpful to specify the version in actionpack and activesupport as well as in the railties.

Reply:
I've updated the PR :+1: